control_plane/attachment_service/src/http.rs

```diff
@@ -451,6 +484,10 @@ pub fn make_router(
         .put("/v1/tenant/:tenant_id/location_config", |r| {
             tenant_service_handler(r, handle_tenant_location_config)
         })
+        .put(
+            "/v1/tenant/:tenant_shard_id/time_travel_remote_storage",
```

Comment:
Param here should be `:tenant_id` -- when the storage controller is impersonating the pageserver API, it always takes requests by tenant ID rather than shard.  `handle_tenant_time_travel_remote_storage` already looks for a "tenant_id" param, so I'm guessing it doesn't work currently.